control-plane/subcommand/connect-init/command.go

```diff
@@ -113,52 +105,47 @@ func (c *Command) Run(args []string) int {
 			return 1
 		}
 	}
-	cfg := api.DefaultConfig()
-	cfg.Namespace = c.flagConsulServiceNamespace
-	c.http.MergeOntoConfig(cfg)
-	consulClient, err := consul.NewClient(cfg, c.http.ConsulAPITimeout())
+
+	// Create Consul API config object.
+	consulConfig := c.consul.ConsulClientConfig()
+
+	// Create a context to be used by the processes started in this command.
+	ctx, cancelFunc := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+	defer cancelFunc()
+
+	// Start Consul server Connection manager
```

Comment:
```suggestion
	// Start Consul server Connection manager.
```
Just to make the all-powerful linter happy.